src/Microsoft.DotNet.ExecutablePackageObtainer/ICanAddPackageToProjectFile.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+
+using Microsoft.Extensions.EnvironmentAbstractions;
+
+namespace Microsoft.DotNet.ExecutablePackageObtainer
```

Comment:
package obtainer has dependency on dotnet.dll's commands, however it should be used by dotnet.dll, so these 2 interface is to break the circular dependency